src/main/java/com/igot/cb/transactional/service/RequestHandlerServiceImpl.java

```diff
@@ -0,0 +1,105 @@
+package com.igot.cb.transactional.service;
+
+import com.fasterxml.jackson.core.JsonProcessingException;
+import com.fasterxml.jackson.core.type.TypeReference;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import com.fasterxml.jackson.databind.SerializationFeature;
+import com.igot.cb.pores.util.Constants;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.http.HttpEntity;
+import org.springframework.http.HttpHeaders;
+import org.springframework.http.HttpMethod;
+import org.springframework.http.MediaType;
+import org.springframework.stereotype.Service;
+import org.springframework.util.CollectionUtils;
+import org.springframework.web.client.HttpClientErrorException;
+import org.springframework.web.client.RestTemplate;
+
+import java.util.HashMap;
+import java.util.Map;
+@Service
+public class RequestHandlerServiceImpl {
+    private Logger log = LoggerFactory.getLogger(RequestHandlerServiceImpl.class);
```

Comment:
If Request Handler intent is to GET, POST request for the platform. Kindly check for reference and try to create parallel service in different folder.

Reply:
we can remove this as of now we are not making any service calls as of now, will be using this further cases.